qemu/tests/blockdev_full_backup_simple.py

```diff
@@ -1,6 +1,13 @@
 from provider.blockdev_full_backup_base import BlockdevFullBackupBaseTest
 
 
+class BlockdevFullBackupBase(BlockdevFullBackupBaseTest):
+    def prepare_test(self):
+        if self.params.get("not_preprocess") == "yes":
+            self.preprocess_data_disks()
+        super(BlockdevFullBackupBase, self).prepare_test()
+
+
```

Comment:
No need to update this py file, pls refer to the comments in blockdev_full_backup.cfg